server/routes/api/representatives.js

```diff
@@ -31,7 +31,7 @@ const STORAGE = buildStorage({
     CACHE.delete(key)
```

Comment:
Changing/cleaning up the representatives route is not related to the donation functionality and should be a different PR. It is best practice to keep PRs very small and focused.

Reply:
Thanks for the feedback. I was secretly hoping changes in `representaties.js` could be lumped together with this PR without creating another issue, but I agree I should've kept the changes in `representatives.js` separate. I'll definitely try the "draft PR" next time to elicit preliminary feedback! Shall I go ahead and revert those commits?

Reply:
Yes, got ahead and revert this part. You can work on improving this as your next issue. If you've never used `git cherry-pick`, this could be a good opportunity to mess with it. Let me know if you need help.

https://git-scm.com/docs/git-cherry-pick